web_src/css/features/projects.css

```diff
@@ -82,12 +83,18 @@
 .card-attachment-images img {
   display: inline-block;
   max-height: 50px;
+  max-width: 100%;
   border-radius: var(--border-radius);
   margin-right: 2px;
```

Comment:
```suggestion
  margin-right: 2px;
  object-fit: contain;
```

Better to have this just in case, to ensure the image always scales equally in both direction (default is `fill`). Might even make this a global rule for all `img`.

Reply:
Would this overlap be a problem?

https://jsfiddle.net/fmapw0ye/


![image](https://github.com/go-gitea/gitea/assets/2114189/279b9516-f782-4d20-bab6-6aff37fb18d5)

Reply:
This will fix that overlap, does require `max-height` on the parent. Better would be direct `max-height` on the `img` without any parent.

```css
max-width: 100%;
max-height: 100%;
aspect-ratio: 1;
```